### DISCUSSION TITLE
Google Maps Beta/Alpha features

### DISCUSSION THREAD

**Discussion by Kilian-Sosa** [nuxt/scripts#245] [Q&A]
Hi! I'm starting using custom markers and noticed that the AdvancedMarkerElements are not clickable by default and you need to enable the option as it is a beta feature. My question is if it is possible to do that with this package or if it will be possible in the future.

Thanks!

**Comment by harlan-zw** [comment]
You can opt into the beta channel via the `v` option:

```ts
useScriptGoogleMaps({ apiKey: '...', v: 'beta' })
```

Advanced markers / clickability options are then available via the standard Maps API. Closing 🙏
